src/routing_table/mod.rs

```diff
@@ -300,9 +301,10 @@ impl<T: Binary + Clone + Copy + Debug + Default + Hash + Xorable> RoutingTable<T
 
     pub fn is_in_our_group(&self, name: &T) -> bool {
         if self.our_group_prefix.matches(name) {
-            return unwrap!(self.groups.get(&self.our_group_prefix)).contains(name);
+            unwrap!(self.groups.get(&self.our_group_prefix)).contains(name)
```

Comment:
Yeah - I'm happy with whatever.  I guess I'm more concerned with consistency, and _always_ favouring early returns is an easy rule to describe and apply, but I don't feel strongly either.  We could either go for that, or what about we leave it up to the author to decide for short `else` blocks, but don't change existing code to suit our own preference?